SwissTransferFeatures/SettingsView/SettingsView.swift

```diff
@@ -16,35 +16,117 @@
  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+import InfomaniakCore
 import InfomaniakDI
 import STCore
+import STResources
 import SwiftUI
 import SwissTransferCore
+import SwissTransferCoreUI
+
+/// Links used in the settings view
+enum SettingLinks {
+    static let discoverInfomaniak = URL(string: "https://www.infomaniak.com/en/about")!
+    static let shareYourIdeas =
+        URL(string: "https://feedback.userreport.com/f12466ad-db5b-4f5c-b24c-a54b0a5117ca/#ideas/popular")!
+}
 
 public struct SettingsView: View {
-    @LazyInjectService var settingsManager: AppSettingsManager
+    @LazyInjectService private var settingsManager: AppSettingsManager
+
+    @EnvironmentObject private var mainViewState: MainViewState
 
-    @StateObject var appSettings: FlowObserver<AppSettings>
+    @StateObject private var appSettings: FlowObserver<AppSettings>
 
     public init() {
         @InjectService var settingsManager: AppSettingsManager
         _appSettings = StateObject(wrappedValue: FlowObserver(flow: settingsManager.appSettings))
     }
 
     public var body: some View {
-        VStack {
-            Text("SettingsView")
-            if let appSettings = appSettings.value {
-                Text(appSettings.theme.name)
+        List(selection: $mainViewState.selectedDestination) {
+            Section(header: Text(STResourcesStrings.Localizable.settingsCategoryGeneral)) {
+                SettingsCell(title: STResourcesStrings.Localizable.settingsOptionTheme,
+                             subtitle: appSettings.value?.theme.title ?? "",
+                             leftIconAsset: STResourcesAsset.Images.brush) {
+                    EditSettingView(Theme.self,
+                                    selected: appSettings.value?.theme ?? .system,
+                                    title: STResourcesStrings.Localizable.settingsThemeTitle) { theme in
+                        @InjectService var settingsManager: AppSettingsManager
+                        _ = try? await settingsManager.setTheme(theme: theme)
```

Comment:
question: Do we want to add a function to a `SettingSelectable` to update the value?
So we could just call the method here?